src/android/Capture.java

```diff
@@ -234,6 +246,17 @@ private void captureAudio(Request req) {
           try {
               Intent intent = new Intent(android.provider.MediaStore.Audio.Media.RECORD_SOUND_ACTION);
 
+              String timeStamp = new SimpleDateFormat("yyyyMMddHHmmssSSS").format(new Date());
+              String fileName = "AUDIO_" + timeStamp + ".wav";
+              File audio = new File(getTempDirectoryPath(), fileName);
+
+              Uri audioUri = FileProvider.getUriForFile(this.cordova.getActivity(),
+                      this.applicationId + ".cordova.plugin.mediacapture.provider",
+                      audio);
+              this.audioAbsolutePath = audio.getAbsolutePath();
+              intent.putExtra(android.provider.MediaStore.EXTRA_OUTPUT, audioUri);
+              intent.addFlags(Intent.FLAG_GRANT_WRITE_URI_PERMISSION);
```

Comment:
why this `intent.addFlags(Intent.FLAG_GRANT_WRITE_URI_PERMISSION);`?

- the Android training https://developer.android.com/training/camera/photobasics#TaskPath does not have this line
- seems to run fine without it in my test app